src/pybind/mgr/rbd_support/module.py

```diff
@@ -108,21 +108,21 @@ def mirror_snapshot_schedule_remove(self,
     @CLIReadCommand('rbd mirror snapshot schedule list')
     @with_latest_osdmap
     def mirror_snapshot_schedule_list(self,
-                                      level_spec: str) -> Tuple[int, str, str]:
+                                      level_spec: Optional[str] = None) -> Tuple[int, str, str]:
```

Comment:
nit: is there a special reason you don't want `Optional[str] = ""`?

Reply:
No, `Optional[str] = ""` should be much better!

Reply:
Actually, `Optional[str] = ""` is too lax.  Since an explicit `None` isn't allowed, it should be just `level_spec: str = ""` (i.e. a regular optional parameter).